### ISSUE TITLE
关于finetune

### ISSUE THREAD

**Issue by DarrenZhu1103** [FlagOpen/FlagEmbedding#497]
您好，我想将bge-m3用于视频检索场景，构造hard negative的方式是用原始的m3模型的dense embedding召回与query最相似标题，通过gpt-4做打标，选取标注结果不相关的样本，目前构造了80k的[query, pos, neg]样本像用于finetune
请问下finetune有推荐的参数设置么，比如batch_size，learning rate，temperature，看到原文里好像没提到

**Comment by staoxiao** [comment]
参考bgev1.5的微调参数就好：https://github.com/FlagOpen/FlagEmbedding/tree/master/examples/finetune#3-train
batch_size尽量大一些，learning rate=1e-5 or 5e-6， temperature=0.02

**Comment by natlank** [comment]
兄弟能问一下你是用Openai的api判断query召回的topk是Positive还是negative吗，你用gpt判断之后还自己筛选一遍吗？我筛选的时候不知道怎么把握区别pos/hard neg的尺度，很烦，能问下你怎么做的吗？还有最后微调效果怎么样呀

**Comment by DarrenZhu1103** [comment]
我们有比较清晰的业务定义，整体看过，感觉还行
微调效果还不错的

**Comment by natlank** [comment]
谢了兄弟
